### DISCUSSION TITLE
Here's what's new in 1.1.2!

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#134] [Announcements]
2 new more menu options

- Feedback
- Github

Icon of inspect tab button changed to a code icon.

**Comment by jdev082** [comment]
Unlocking conversation once the release releases!

**Reply by jdev082** [reply]
Released!

**Comment by imdevarsh** [comment]
what else are you planning?

**Reply by jdev082** [reply]
@webdev03 whatever features a one developer can make on his own :(
